src/main/markdown/doc/latest/DevGuideI18nLocale.md

```diff
@@ -76,9 +76,12 @@ are probably better off choosing some real language as a default, such as
 the following in your [module XML](DevGuideOrganizingProjects.html#DevGuideModuleXml):
 
 ```
-<set-property-fallback name="locale" value="en"/>
+<set-property-fallback name="locale" value="en"/> <!-- 1. fallback locale -->
+<set-property name="locale" value="en"/> <!-- 2. full list of supported locales-->
```

Comment:
Maybe give an example with more than one value here (e.g. "en, es"), along with the corresponding `extend-property`.

Reply:
do we really need to extend the property with the full list before setting it after the fallback?

Reply:
Yes.

Here, I included only the `set-property-fallback` and `set-property`:
```
[ERROR] The fallback value 'en' was not previously defined for property 'locale'
```
Then I added `<extend-property name=locale" values="en"/>`:
```
[ERROR] Line 36: Value 'es' in not a valid value for property 'locale'
```

The full working snippet is:
```xml
<extend-property name="locale" values="en,es"/>
<set-property name="locale" value="en,es"/>
<set-property-fallback name="locale" value="en"/>
```

It doesn't matter whether `set-property-fallback` is done before or after the `set-property`, so having `set-property` next to `extend-property` with the same values makes things easier to maintain (IMO).